Azure/sap-automation
# Minor updates

## Problem
More validation steps

- Check for correct Oracle distros
- Check for disk space on deployer

## Solution
<Please elaborate the solution for the problem>

## Tests
<Please provide steps to test the PR>

## Notes
<Additional comments for the PR>
